scene/main/timer.cpp

```diff
@@ -116,16 +116,20 @@ void Timer::stop() {
 	autostart = false;
 }
 
-void Timer::set_active(bool p_active) {
-	if (active == p_active)
+void Timer::set_paused(bool p_paused) {
+	if (paused == p_paused)
 		return;
 
-	active = p_active;
+	paused = p_paused;
 	_set_process(processing);
 }
 
-bool Timer::is_active() const {
-	return active;
+bool Timer::is_paused() const {
+	return paused;
+}
+
+bool Timer::is_running() const {
+	return get_time_left() > 0;
```

Comment:
Although, apparently the time left can never be 0 while the timer is running, I would do `time_left != -1` instead. Also, could `is_running` be confusing? It may return true even if the timer is paused, so maybe `stopped` is a better name.
Just my opinion. PR looks good.